platform/src/main/resources/assets-test.json

```diff
@@ -122,6 +123,7 @@
                     }
                 ]
             },
+            "sep24_enabled": true,
```

Comment:
Do you really mean to enable both USD and USDC for SEP-24? I thought this SEP actually means the bridge between these assets.

Here's how the SEP-24 info is looking like: https://anchor-sep-pr218.previews.kube001.services.stellar-ops.com/sep24/info

Reply:
You are right. It should be USDC only. 

`sep24_enabled` in USD is false now.